test/test_model.js

```diff
@@ -1924,7 +1927,8 @@ describe('test_model.js', function() {
       db.units.add(units, true);
       var placement = db._mapServicesToMachines(db.machines);
```

Comment:
this is a case of testing the private methods directly. It's good sometimes, but just pointing out as we talked about this today.